bokeh/models/widgets/inputs.py

```diff
@@ -89,26 +87,25 @@ def coerce_value(cls, val):
 #-----------------------------------------------------------------------------
 
 class FileInput(Widget):
-    ''' Present a file-chooser dialog to users and return the contents of a
-    selected file.
+    ''' Present a file-chooser dialog to users and return the contents of the 
+    selected files 
 
     '''
-
-    value = String(default="", readonly=True, help="""
-    A base64-encoded string of the contents of the selected file.
-    """)
-
-    mime_type = String(default="", readonly=True, help="""
-    The mime type of the selected file.
-    """)
-
-    filename = String(default="", readonly=True, help="""
-    The filename of the selected file.
-
-    .. note::
-        The full file path is not included since browsers will not provide
-        access to that information for security reasons.
-    """)
+    
+    value = Either(String, List(String),default='', readonly=True, help='''
```

Comment:
spaces after comma before `default` to match existing style also below)

Reply:
No problem, I can do that.

Reply:
Done in PR #9732